conditionals.py

```diff
@@ -132,7 +145,13 @@
 # Your challenge is to translate the steps above into conditionals which will evaluate if the 
 # year stored in a variable is/was a leap year.
 
-
+year = int(input("Enter a year:\n"))
+if year%4 == 0:
+   if year%100 == 0:
+      if year%400 == 0: 
+   print("This is a leap year!")
```

Comment:
This script does not run because of an Indentation Error here. Make sure the `print("")` is properly indented beneath the last if statement. 
```suggestion
           print("This is a leap year!")
```